### DISCUSSION TITLE
Coverage Map/Map Painting

### DISCUSSION THREAD

**Discussion by coatsy35** [Courseplay/Courseplay_FS22#2256] [Ideas]
It would be good to have the minimap/large map colour in the field as the application is applied just like a real agricultural GPS system does. It is called a coverage map. Usually this is a yellow colour and is painted as your travel with AI/Steering engaged. This allows you to tell where you application has stopped i.e. if the sprayer runs out of chemical etc.... You would then be able to reset/clear the coverage log at any point, if you perform another action over the existing log in a real system it would change the shade so you can see where you have been.

I think  a single coverage log would be sufficient for the game which would be resettable on the HUD. Not sure if this idea is even possible but it would be a fantastic addition - unless someone is going to tell me it already exists! Basically this would work just like the yield paints the map when harvesting with the precision farming addon.

**Comment by Tensuko** [comment]
There are two ways to get what you want:
1) let CP start at the last Waypoint, it will automatically drive there
2) Turn on "eye Icon" on the HUD to its blue state, wich will show the 20 next and the 5 last waypoints.

For the Minimap, we can think of some marker to show the last waypoint.

**Reply by coatsy35** [reply]
That would be good. Really good if you can colour in where you have been on the map! and a marker for last place on the minimap and on the field would be excellent. I was spraying the other day and had to stop get more chemical I pressed resume from last waypoint but it wouldn't find it, just wanted to start the course again. At least with a marker or colouring you can drive roughly to where you should be and then resume form there.
